### ISSUE TITLE
Can this work with the RetroAcrch UWP for xbox dev mode?

### ISSUE THREAD

**Issue by danprice142** [gfx-rs/portability#248]

**Comment by kvark** [comment]
I haven't tried any UWP applications, but gfx-portability can certainly help by routing things to DX12.

**Comment by danprice142** [comment]
I dont know a lot about all this but to my understanding Xbox One has ANGLE/DirectX11 Series console have ANGLE/DirectX12 having higher the ability to use Vulkan and higher versions of OpenGL which would add so much more functionality to RetroArch as all Libretro Cores could then be used. Would you or anyone in the team attempt something like this? RetroArch is also open source and i can help out with some testing in dev mode but im not a coder. its here if you would like to take a look https://github.com/libretro/RetroArch/tree/master/uwp

**Comment by kvark** [comment]
I'd be happy to give it a shot at some point, but for the nearest future my schedule got very busy with WebGPU things.
Other people are welcome to try this out!

**Comment by JustMeDaFaq** [comment]
Yes, there's a quick and dirty done poc implementstion utilizing that library in retroarch. I need some time to do anything proper that would make it to the official repo.

**Comment by danprice142** [comment]
This sounds great guys, ill be keeping a eye on this. Thanks so much and ill be available for testing if that's any help, definingly keeping my fingers crossed

**Comment by danprice142** [comment]
@JustMeDaFaq Anymore progress on this or anything open source yet?
